controllers/errors.controller.js

```diff
@@ -0,0 +1,11 @@
+exports.handleCustomErrors = (req, res, next) => {
+    if (err.status) {
+        res.status(err.status).send({ msg: err.msg });
+    } else {
+        next(err);
+    }
+};
```

Comment:
This shouldnt need to be added yet as we dont have any tests written to ensure this works as expected.